src/test/java/com/ss/greencity/CommentsTest.java

```diff
@@ -0,0 +1,68 @@
+package com.ss.greencity;
+
+import com.ss.greencity.pageobjects.*;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+public class CommentsTest extends EcoNewsTestRunner {
+
+    @Test
+    public void addComment() {
+        System.out.println("Sign in");
+        SignInPO signInPO = new SignInPO(driver);
+        signInPO.clickSignInButton()
+                .setEmail("aliejua@gmail.com")
+                .setPassword("Aa12345_")
+                .clickSignInUserButton();
+
+        System.out.println("Go to Eco News, open the first news card");
+        HeaderSignedInPO headerSignedInPO = new HeaderSignedInPO(driver);
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        headerSignedInPO.getNews().click();
+        EcoNewsListPO newsListPO = new EcoNewsListPO(driver);
+        newsListPO.getFirstNewsCard().getClickableArea().click();
+
+        System.out.println("Add a new comment");
+        CommentsPO commentsPO = new CommentsPO(driver);
+        String timeNow = new SimpleDateFormat("yyyyMMdd_HHmmss")
+                .format(Calendar.getInstance().getTime());
+        commentsPO.getCommentsInputbox().setData(timeNow);
+        commentsPO.getCommentButton().click();
+        try {
+            Thread.sleep(15000);
```

Comment:
Do you really need waiting for so long?

Reply:
@Meownjik, yeah, otherwise, the next step will fail because of not finding just added comment.